packages/marketing-modal/src/MarketingModal.tsx

```diff
@@ -178,6 +191,10 @@ const MarketingModal = ({
       darkMode={darkMode}
       closeIconColor={closeIconColor}
     >
+      {!darkMode &&
+        showBlob &&
+        graphicStyle === GraphicStyle.Center &&
```

Comment:
Why only when GraphicStyle is Center?

Reply:
The designs only show the blob when the graphic is centered. `Fill` is when the image is 100% width of the container so my guess is the modal might become too busy with the blob and the full width image.

Reply:
That's fair, just wanted to double check